README.md

```diff
@@ -1409,16 +1409,16 @@ Trivy scans a tar image with the following format.
   - Kaniko (https://github.com/GoogleContainerTools/kaniko)
 
 
-### Data source
-- PHP
+### Data Sources
+- PHP Security Advisories Database
   - https://github.com/FriendsOfPHP/security-advisories
-- Python
+- Safety DB (Python)
   - https://github.com/pyupio/safety-db
-- Ruby
+- Ruby Advisory Database
   - https://github.com/rubysec/ruby-advisory-db
-- Node.js
+- Node.js Security Working Group
   - https://github.com/nodejs/security-wg
-- Rust
+- RustSec Advisory Database
   - https://github.com/RustSec/advisory-db
```

Comment:
I want to keep it as is because I'll add more sources to each language.

- PHP
  - https://github.com/FriendsOfPHP/security-advisories
  - https://example.com/
  - https://example2.com/